recipes/elfio/all/test_package/conanfile.py

```diff
@@ -0,0 +1,17 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class ElfioTestConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake"
```

Comment:
```suggestion
    generators = "cmake", "cmake_find_package_multi"
```

Reply:
why adding the cmake_find_package_multi to the test, will this ever be used?

Reply:
yes, by find_package

Reply:
no, because we do not include the multi file in the cmake file, 
afaik this needs to query of a multi generator is used, and then include an other file
I do not think that adds any value to the test of the package

Reply:
config files are not "included", they are discovered by find_package. This modification is required if you accept the next modification in test_package.